includes/polling/os/fs-nmu.inc.php

```diff
@@ -0,0 +1,42 @@
+<?php
```

Comment:
you should move this to the new format, see for example, https://github.com/librenms/librenms/pull/11932

Reply:
@Jellyfrog to this function public `function discoverOS(): void` right ? I will move it

Reply:
Yep yep

Reply:
@Jellyfrog moved to new format